scripts/docker/install-shell-tools.bash

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Copyright 2019 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+wget https://iterm2.com/utilities/it2check -P /usr/local/sbin/
```

Comment:
I am afraid that users cannot run iTerm2 from without a Docker container? If I am correct, iTerm2 users would have to download the it2check script on their mac computers manually?  If so, how about we include the script it2check as a string constant of repl, and call `os/exec.Cmd` to run the script?

Reply:
1. A user can use iTerm2 to start the Docker container and run REPL in the container, so she doesn't need to download the it2check script manually. 
1. REPL can only run in a Docker container at the moment because it depends on SQLFlow core. Later we may try to extract REPL to a standalone repo and make it communicate to SQLFlow server via GRPC rather than call SQLFLow core directly, thus decoupling it from Docker. We don't need to include `it2check` in the binary until then.